lidar/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+open3d
+laspy
+pyproj
```

Comment:
please add lazrs otherwise ou get the folowing error calling `pc = load_data(datadir, 1000)` : 

`LaspyException: No LazBackend selected, cannot decompress data
Output is truncated. View as a scrollable element or open in a text editor. Adjust cell output settings...`

Reply:
Indeed I forgot that the full laspy install is `pip install laspy[lazrs,laszip]`, fixed now!